modules/ROOT/pages/api-led-test.adoc

```diff
@@ -64,60 +63,61 @@ http://localhost:8081/api/greeting
 ----
 
 . Return to Studio and notice that there's a dashed line around *Set Payload*.
-. In the canvas, click *Set Payload* to open the *Set Payload* node in the *Mule Debugger* above the canvas. Open *Set Payload* and see that the payload value is `Hi.`, three characters.
-. Click the curved yellow arrow above the debugger window.
+. In the canvas, click *Set Payload* to open the *Set Payload* node in the *Mule Debugger* above the canvas. 
+. Open *Set Payload* and see that the payload value is `Hi.`, three characters.
+. Click the *Next processor* (curved yellow arrow) above the debugger window.
```

Comment:
I guess you need the aboves

Reply:
In this case, I didn't see a way to avoid it.